libmachine/provision/boot2docker.go

```diff
@@ -188,8 +180,8 @@ func (provisioner *Boot2DockerProvisioner) Provision(swarmOptions swarm.SwarmOpt
 	return nil
 }
 
-func (provisioner *Boot2DockerProvisioner) SSHCommand(args ...string) (*exec.Cmd, error) {
-	return drivers.GetSSHCommandFromDriver(provisioner.Driver, args...)
+func (provisioner *Boot2DockerProvisioner) SSHCommand(args string) (*ssh.Output, error) {
+	return drivers.GetSSHCommandFromDriver(provisioner.Driver, args)
```

Comment:
This is way too complicated. `SSHCommand` calls `GetSSHCommandFromDriver` which probably actually runs some command but none of these have the word `Run` in it.

Also there are few other occurrences of this one-line methods. I don't think we really need those and they're not really consistently named across drivers.

Also why don't we can't just do `provisioner.Driver.RunSSHCommand(args)` instead of going back to the `drivers` package? Sounds like we can do better.